tests/test_upgrade_shared.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+
+from helpers import run_pipx_cli
+
+
+def test_upgrade_shared(pipx_ultra_temp_env, capsys, caplog, monkeypatch):
```

Comment:
```suggestion
def test_upgrade_shared(pipx_ultra_temp_env, capsys, caplog):
```

This doesn't seem to be used in the test?